src/engines/kotor/gui/ingame/container.cpp

```diff
@@ -53,11 +53,15 @@ void ContainerMenu::fillFromInventory(const Inventory &inv) {
 	const std::map<Common::UString, InventoryItem> &invItems = inv.getItems();
 	for (std::map<Common::UString, InventoryItem>::const_iterator i = invItems.begin();
 			i != invItems.end(); ++i) {
-		Item item(i->second.tag);
-		lbItems->addItem(Common::UString::format("%s|%s|%u",
-		                                         item.getName().c_str(),
-		                                         item.getIcon().c_str(),
-		                                         i->second.count));
+		try {
+			Item item(i->second.tag);
+			lbItems->addItem(Common::UString::format("%s|%s|%u",
+													 item.getName().c_str(),
+													 item.getIcon().c_str(),
+													 i->second.count));
+		} catch (...) {
+			warning("Failed to load item %s", i->second.tag.c_str());
```

Comment:
If catch a `Common::Exception &e`, you can use add() on that to add that message and use `Common::printException(e, "WARNING: ")`, for a more informative message.